tests/check_licenses.go

```diff
@@ -30,7 +30,11 @@ func main() {
         ".github",
         ".peg", // exlude all .peg.go files
         "build",
-        "third_party",
+		"third_party",
+		"crypto", // boringssl source code files
+		"ssl",
+		"util",
+		"sources.cmake",
```

Comment:
It looks like the indentation is off here.

Reply:
Existing code uses 4 white spaces but this PR uses tab, which is default indentation used by `go fmt` (in GitHub UI without special configuration, tab seems 8 or more white spaces).  
I just use `go fmt` to format the whole file.
https://stackoverflow.com/questions/19094704/indentation-in-go-tabs-or-spaces